var/spack/repos/builtin/packages/alpaka/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Alpaka(CMakePackage):
+    """Abstraction Library for Parallel Kernel Acceleration."""
+
+    homepage = "https://alpaka.readthedocs.io"
+    url      = "https://github.com/alpaka-group/alpaka/archive/refs/tags/0.6.0.tar.gz"
+    git      = "https://github.com/alpaka-group/alpaka.git"
+
+    maintainers = ['vvolkl']
+
+    version('develop', branch='develop')
+    version('0.6.0', sha256='7424ecaee3af15e587b327e983998410fa379c61d987bfe923c7e95d65db11a3')
+
+    variant("acc_cpu_b_seq_t_seq", default=True, description="Enable the serial CPU back-end")
+    variant("acc_cpu_b_seq_t_threads", default=True,
+            description="Enable the threads CPU block thread back-end")
+    variant("acc_cpu_b_seq_t_fibers", default=False, description="Enable the fibers CPU block thread back-end")
+    variant("acc_cpu_b_tbb_t_seq", default=False, description="Enable the TBB CPU grid block back-end")
+    variant("acc_cpu_b_omp2_t_seq", default=False, description="Enable the OpenMP 2.0 CPU grid block back-end")
+    variant("acc_cpu_b_seq_t_omp2", default=False, description="Enable the OpenMP 2.0 CPU block thread back-end")
+    variant("acc_any_bt_omp5", default=False, description="Enable the OpenMP 5.0 CPU block and block thread back-end")
+    variant("acc_any_bt_oacc", default=False, description="Enable the OpenACC block and block thread back-end")
```

Comment:
Two comments:
1. Would it make sense to use a multi-valued variant?
2. Can we use names not tied 1:1 with the CMake option?

Just wondering if we can encode something like:
```
$ spack install alpaka backend=serial,fibers,omp5
```
rather than:
```
$ spack install alpaka +acc_cpu_b_seq_t_seq +acc_cpu_b_seq_t_fibersacc_cpu_b_seq_t_fibers +acc_any_bt_omp5
```

Reply:
Yes, that's definitely more elegant, I'll change it. Thanks!

Reply:
There are now two variants that are exclusive, i.e. if set all others should be unset. I added some conflicts, not sure if there is a better way